jenkins-integration/jenkins-jobs/common/scripts/background/curl-server-metrics-loop.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -x
+
+while true ; do
```

Comment:
For dev run purposes, we may want to nuke this file, if it exists, before starting the `curl` loop - just so that we don't end up with status info left over from a prior job.